requirements/sentry.in

```diff
@@ -0,0 +1,3 @@
+# Extra requirements
+
```

Comment:
Could you add an explanation on why this version is pinned?

Maybe it could be:

# pinning sentry-sdk version in order to avoid the usage of non-compatible version of sentry

Reply:
Sure, no problem.